src/main/java/com/courtcanva/ccfranchise/controllers/StaffController.java

```diff
@@ -0,0 +1,25 @@
+package com.courtcanva.ccfranchise.controllers;
+
+import com.courtcanva.ccfranchise.services.StaffService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Email;
+
+@RestController
+@RequestMapping("/staffs")
+@RequiredArgsConstructor
+@Validated
+public class StaffController {
+    private final StaffService staffService;
+
+    @GetMapping("/email/exists")
```

Comment:
We do not recommend using verbs in restful API. ` Head` method is an option for checking if a resource exists.

Reply:
Really good suggestion for me. I have modified the code, please check again.